### ISSUE TITLE
unknown flag: --skip-steps

### ISSUE THREAD

**Issue by gendolf3d** [argoproj/argo-rollouts#469]
Hello All!

I would like to promote Canary Rollout and trying to execute the following command in accordance with documentation:
`kubectl argo rollouts promote myservice -n mynamespace --skip-steps`
However, it yields:
`Error: unknown flag: --skip-steps`

Looks like a bug either in Documentation or kubectl plugin

Plugin documentation:
```
kubectl argo rollouts promote --help
Promote a rollout

Usage:
  kubectl-argo-rollouts promote ROLLOUT [flags]

Examples:
	# Promote a rollout with the canary strategy
	kubectl argo rollouts promote guestbook [--skip-steps] [--skip-current-step]

Flags:
      --as string                      Username to impersonate for the operation
      --as-group stringArray           Group to impersonate for the operation, this flag can be repeated to specify multiple groups.
      --cache-dir string               Default HTTP cache directory (default "/Users/adurandin/.kube/http-cache")
      --certificate-authority string   Path to a cert file for the certificate authority
      --client-certificate string      Path to a client certificate file for TLS
      --client-key string              Path to a client key file for TLS
      --cluster string                 The name of the kubeconfig cluster to use
      --context string                 The name of the kubeconfig context to use
  -h, --help                           help for promote
      --insecure-skip-tls-verify       If true, the server's certificate will not be checked for validity. This will make your HTTPS connections insecure
  -v, --kloglevel int                  Log level for kubernetes client library
      --kubeconfig string              Path to the kubeconfig file to use for CLI requests.
      --loglevel string                Log level for kubectl argo rollouts (default "info")
  -n, --namespace string               If present, the namespace scope for this CLI request
      --request-timeout string         The length of time to wait before giving up on a single server request. Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h). A value of zero means don't timeout requests. (default "0")
  -s, --server string                  The address and port of the Kubernetes API server
  -a, --skip-all-steps                 Skip remaining steps
  -c, --skip-current-step              Skip current step
      --token string                   Bearer token for authentication to the API server
      --user string                    The name of the kubeconfig user to use
```

**Comment by jessesuen** [comment]
Documentation looks wrong. The flags are:
```
  -a, --skip-all-steps                 Skip remaining steps
  -c, --skip-current-step              Skip current step
```

**Comment by cronik** [comment]
The example was corrected in `0.8.0`, see #454

https://argoproj.github.io/argo-rollouts/generated/kubectl-argo-rollouts/kubectl-argo-rollouts_promote/

**Comment by jessesuen** [comment]
Fixed
